test.p4

```diff
@@ -0,0 +1,8 @@
+Checking if pruned/ exists.
```

Comment:
What"s this?

Reply:
Sorry i was just debugging stuff, i'll remove this